### DISCUSSION TITLE
Scrollbar for thumbnail dots do not work

### DISCUSSION THREAD

**Discussion by milelazar** [dimsemenov/PhotoSwipe#2076] [Q&A]
I have image thumbnails as gallery pagination. When thumbs do not fit into screen, scrollbar appears. But I am unable to scroll with mouse wheel.

wheelToZoom for image zoom is enabled but there is no difference if disabled. Is there a way to enable scrolling for that section?. 

I have tried to add a filter like this but it does not work:
```
lightbox.addFilter('preventPointerEvent', (preventPointerEvent, originalEvent, pointerType) => {
  if (originalEvent.target.closest('.pswp__bullets-indicator')) {
    return false;
  }
  return true;
});
```

Here is the example screenshot:

![Screenshot 2023-11-24 at 14 56 33](https://github.com/dimsemenov/PhotoSwipe/assets/1022682/44ba462d-e2d0-4242-99a6-9b7d96799797)

**Comment by cuongdaoict** [comment]
![Screenshot_11](https://github.com/dimsemenov/PhotoSwipe/assets/160107501/948f3896-6713-43a1-9e3e-5758d1509d47)
I can't use mouse for scroll content on this.

**Comment by BilouMaster** [comment]
I've had the same issue and solved it like this:

```js
function enableScroll(e){
  let mst = e.target.scrollHeight - e.target.clientHeight;
  if (mst != 0) {
    pswp.options.wheelToZoom = false;
    e.target.scroll({
      top: e.deltaY, behavior: "smooth"
    });
    setTimeout(function() {pswp.options.wheelToZoom = true;}, 100);
  }
}

// somewhere else, with "el" as the scrollable element (just after its creation):
el.addEventListener('wheel', enableScroll);
```
